app/core/services/ProcessPopulation.scala

```diff
@@ -103,4 +116,27 @@ trait ProcessPopulation {
           case Left(err) => Left(err)
         }
     }
+
+  private def processSequenceOptions(id: String, options: Seq[Phrase]): Either[GuidanceError,(Seq[Phrase], Boolean)] = {
```

Comment:
this could be simplified to just detect use of exclusive with the main error handling done from PageBuilder.pagesWithValidation(), where the error validation will be added in EG-1313

Reply:
Done